src/browser/AppInit.tsx

```diff
@@ -66,6 +66,7 @@ applyKeys(
   'documents',
   'folders',
   'grass',
+  'guides',
```

Comment:
Unfortunately we can't serialize the guides like this, as they contain Jsx elements which don't serialize properly and cause a crash when you try to view guides after reloading...